examples/tool_chat_template_llama3.2_json.jinja

```diff
@@ -120,7 +125,7 @@
         {{- "<|eot_id|>" }}
     {%- elif message.role == "tool" or message.role == "ipython" %}
         {{- "<|start_header_id|>ipython<|end_header_id|>\n\n" }}
-        {%- if message.content is mapping or message.content is iterable %}
+        {%- if message.content is mapping %}
```

Comment:
Maybe we can invert the condition? i.e. check whether it is a string like in L32